core/chains/evm/config/toml/defaults/Merlin_Testnet.toml

```diff
@@ -0,0 +1,20 @@
+ChainID = 686868
+ChainType = "zkevm"
+FinalityDepth = 1000
+FinalityTagEnabled = false
+LogPollInterval = "4s" # blockrate is 3s; adding a 1s buffer
```

Comment:
Why do we want to poll more slowly than the block rate? I would expect us to err the other way, by polling faster to reduce latency. Polling slower than the block rate means in some cases there will be two blocks waiting for us by the time we poll again.